app/models/customer.rb

```diff
@@ -0,0 +1,13 @@
+class Customer < ApplicationRecord
+    validates :first_name, presence: true
+    validates :last_name, presence: true
+    validates :phone, presence: true
+    validates :phone, numericality: { only_integer: true}
+    validates :phone, length: { is: 10}
+    validates :email, presence: true
```

Comment:
You could also add a uniqueness validation. This is especially useful to make sure attributes used for login (usually the email) aren't duplicated.